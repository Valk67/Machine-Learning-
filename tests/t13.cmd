10 10 5 8 0 0
