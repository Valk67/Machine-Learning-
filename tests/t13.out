Gxxxxxxxxx
^<.......x
x^.......x
x^.......x
x^.......x
x^.......x
>^........
^xxxxxxxx.
^xxxvSxxx.
^<<<<.....
